### PR TITLE
adjusting image picker html page to be used in pat-raptor

### PR DESCRIPTION
**Pull request by garbas**

### CODE REVIEW

File: feedback/panel-image-picker.html [hunk 1]
```diff
@@ -1,76 +1,89 @@
...
-title: Image picker
-design_title: Image picker panel
-design_description: "Panel that allows users to pick an image"
-buttons:
-  - label: Insert selected image
-    type: submit
-    class: 'close-panel icon-ok-circle'
-  - label: Cancel
-    type: button
-    class: 'close-panel icon-cancel-circle'
```
Comment (jcbrand) [inline feedback/panel-image-picker.html]: @garbas Shouldn't the metadata above be kept in the file? Why was it removed?

**Comment by garbas** [comment]
@jsbrand i've used generic template to have more control over html. to be honest i have not a clear idea where should be what.

File: feedback/panel-image-picker.html (hunk 1 above)
Comment (garbas) [inline feedback/panel-image-picker.html]: @jsbrand i've used generic template to have more control over html. to be honest i have not a clear idea where should be what.

Comment (jcbrand) [inline feedback/panel-image-picker.html]: Well, me neither to be honest. @cornae, could you perhaps chime in?

**Pull request merged**